src/nvme/private.h

```diff
@@ -180,7 +180,7 @@ int json_dump_tree(nvme_root_t r);
 nvme_ctrl_t __nvme_lookup_ctrl(nvme_subsystem_t s, const char *transport,
 			       const char *traddr, const char *host_traddr,
 			       const char *host_iface, const char *trsvcid,
-			       nvme_ctrl_t p);
+			       nvme_ctrl_t p, const char *subsysnqn);
```

Comment:
Isn't the `p` argument used as starting point of iterator? In this case I would swap the order of `p` and `subsysnqn` so that `p` stays at the last position.

Reply:
Done